### PR TITLE
In `*.md` files use 4 spaces wide indent for better compatibility with GitHub and Bitbucket

### PR DESCRIPTION
**Pull request by mbohal**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -12,5 +12,5 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.scss]
+[*.{scss,md}]
```
Comment (adamkudrna) [inline .editorconfig:15]: No reformatting needed…?

**Review by adamkudrna — APPROVED** [review]
Just a tiny comment on the commit message:

```diff
- In *.md files use 4 spaces wide indent for better compatibility with GitHub and Bitbucket
+ Use 4 spaces wide indent in `*.md` files for better compatibility with GitHub and Bitbucket
```

(SVOMPT rule)

or at least with comma:

```diff
- In *.md files use 4 spaces wide indent for better compatibility with GitHub and Bitbucket
+ In *.md files, use 4 spaces wide indent for better compatibility with GitHub and Bitbucket
```

File: .editorconfig (hunk 1 above)
Comment (mbohal) [reply .editorconfig:15]: No, the linter is already set correctly (https://github.com/react-ui-org/react-ui/blob/ea1d90341f80be267e82bb1ac59a3d1982e47e15/.markdownlint.jsonc#L6). This is here to make my IDE use correct indent when I press `Enter`.

**Comment by mbohal** [comment]
I changed the commit message as suggested.

**Pull request merged**